packages/commonwealth/server/util/requirementsModule/validateGroupMembership.ts

```diff
@@ -1,5 +1,7 @@
-import { Requirement } from './requirementsTypes';
+import { AllowlistData, Requirement, ThresholdData } from './requirementsTypes';
 import { TokenBalanceCache } from '../../../../token-balance-cache/src';
+import { ChainNetwork } from '../../../../common-common/src/types';
```

Comment:
Can we make these cross-package imports use absolute rather than relative paths?

Reply:
fixed this